cocos/renderer/pipeline/Define.cpp

```diff
@@ -63,6 +47,38 @@ const gfx::UniformBlock UBOLocalBatched::LAYOUT = {
     1,
 };
 
+const String UBOCamera::NAME = "CCCamera";
+const gfx::DescriptorSetLayoutBinding UBOCamera::DESCRIPTOR = {
+    UBOCamera::BINDING,
+    gfx::DescriptorType::UNIFORM_BUFFER,
+    1,
+    gfx::ShaderStageFlagBit::ALL,
+};
+const gfx::UniformBlock UBOCamera::LAYOUT = {
+    GLOBAL_SET,
+    UBOCamera::BINDING,
+    UBOCamera::NAME,
```

Comment:
`UBOCamera::LAYOUT`, `UBOCamera::BINDING` and `UBOCamera::NAME` are all static member variables. `UBOCamera::BINDING` and `UBOCamera::NAME` may not be initialized when initializing `UBOCamera::LAYOUT`. So i think it is not a good way to use them here.

Reply:
> `UBOCamera::LAYOUT`, `UBOCamera::BINDING` and `UBOCamera::NAME` are all static member variables. `UBOCamera::BINDING` and `UBOCamera::NAME` may not be initialized when initializing `UBOCamera::LAYOUT`. So i think it is not a good way to use them here.

UBOCamera::BINDING is initialized when constructed in the header file, and UBOCamera::NAME is initialized before UBOCamera::LAYOUT is assigned. Or it would have to be set up all at the time of construction in header file.

Reply:
Yep, i think it can work now. But it depends on the sequence, if the sequence changes in future, then it will not work. And it may be hard to find the reason.